Remove dash from token value in swap

tmashuang opened the pull request:
The `swapTokenValue` is coming in with a `-` prefix already during the conversion process. This removes/replaces the `-`.

<details>
  <summary>Before</summary>
    <img src="https://user-images.githubusercontent.com/13376180/97368491-4c422200-1868-11eb-9d1c-891d326d6667.png">
</details>

<details>
  <summary>After</summary>
  <img src="https://user-images.githubusercontent.com/13376180/97368524-5f54f200-1868-11eb-9d9f-06a5c3568f52.png">
</details>

tmashuang committed:
Remove dash from token value in swap

tmashuang committed:
Test removeDash

tmashuang committed:
Lint

Gudahtt commented on ui/app/hooks/useTransactionDisplayData.js:
```diff
@@ -143,7 +143,7 @@ export function useTransactionDisplayData (transactionGroup) {
       ? currentAsset.symbol
       : initialTransaction.sourceTokenSymbol
-    primaryDisplayValue = swapTokenValue
+    primaryDisplayValue = removeDash(swapTokenValue)
```
I think this would give us the wrong answer in any case where the dash exists. It is important to know whether this is a negative value or not.

For example, in this case I encountered:

![plus-minus](https://user-images.githubusercontent.com/2459287/97370823-d99a6c00-1892-11eb-827c-3b4449723bcf.png)

In this case, `isViewingReceivedTokenFromSwap` was `true`, so the prefix would be `+`. But we're _losing_ ETH here, so the correct prefix is `-`.

I think we want to do two things here:

* "reverse" the prefix used (e.g. add or remove a dash, or multiply by `-1`, whichever) if `isViewingReceivedTokenFromSwap` is false, and
* add `+` if the resulting number is positive.

tmashuang replied:
> But we're _losing_ ETH here, so the correct prefix is `-`.

Wait if you swap a token to ETH, we are losing ETH?

Gudahtt replied:
>Wait if you swap a token to ETH, we are losing ETH?

Potentially, yep! In this case, I lost money because the transaction failed on-chain. I had to pay the gas fee but I got nothing in return.

Gudahtt reviewed (changes requested):
Thanks for taking a look at this! We do need to eliminate this extra dash, but I think this would give the wrong answer in some cases.

tmashuang commented:
Also, seems when a swap is pending the `swapTokenValue` is null. Maybe conditional should be used there, but seems that there needs to be a bigger PR that addresses some of the comments.
